### PR TITLE
 use grep to find all a/an typos

### PR DESCRIPTION
**Pull request by allencloud**
use grep to find all a/an typos

### CODE REVIEW

File: docs/admin/logging/fluentd.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ and [its documents](http://docs.fluentd.org/).
 To use this logging driver, start the `fluentd` daemon on a host. We recommend
 that you use [the Fluentd docker
 image](https://hub.docker.com/r/fluent/fluentd/). This image is
-especially useful if you want to aggregate multiple container logs on a each
+especially useful if you want to aggregate multiple container logs on an each
```
Comment (albers) [inline docs/admin/logging/fluentd.md]: either "on a host" or "on each host"

Comment (allencloud) [inline docs/admin/logging/fluentd.md]: I think on each host is better, updated.

**Comment by allencloud** [comment]
@thaJeztah @albers 
Do you know what is the wrong with the **documentation** test?

**Comment by thaJeztah** [comment]
@allencloud I've seen it fail on other PR's, but should not be related, let me try to restart it

LGTM

**Comment by thaJeztah** [comment]
retest this please

**Comment by albers** [comment]
Looks like only builds with "no docs changes" succeed.

**Comment by thaJeztah** [comment]
@albers I pinged @SvenDowideit, who may know more about it

**Comment by SvenDowideit** [comment]
retest this please

File: CHANGELOG.md [hunk 2]
```diff
@@ -1721,7 +1721,7 @@ With the ongoing changes to the networking and execution subsystems of docker te
 + Add -rm to docker run for removing a container on exit
 - Remove error messages which are not actually errors
 - Fix `docker rm` with volumes
-- Fix some error cases where a HTTP body might not be closed
+- Fix some error cases where an HTTP body might not be closed
```
Comment (SvenDowideit) [inline CHANGELOG.md]: oh, what colour is my bike shed! :)

**Comment by SvenDowideit** [comment]
LGTM

**Comment by SvenDowideit** [comment]
retest this please

File: CHANGELOG.md (hunk 2 above)
Comment (thaJeztah) [inline CHANGELOG.md]: heh, yes, still not sure which one is right here as well, both work

**Pull request merged**